interpreter/cling/test/lit.cfg

```diff
@@ -109,6 +108,12 @@ config.substitutions.append(('%shlibext', config.shlibext))
 if platform.system() not in ['Windows'] or lit_config.getBashPath() != '':
     config.available_features.add('shell')
 
+# Filter-out tests not supposed to run on Windows
+if platform.system() not in ['Windows']:
+    config.available_features.add('not_system-windows')
+
```

Comment:
We don't need this either

Reply:
Actually, it was removed as part of https://github.com/root-project/root/commit/e4823b01203e663d6efb1b7b1c9297b8d64f3afc, so it should not be re-introduced.